lib/storage/data/file/DataFileStore.js

```diff
@@ -199,6 +199,16 @@ class DataFileStore {
                 return cbOnce(errors.InternalError.customizeDescription(
                     `read stream error: ${err.code}`));
             });
+            dataStream.on('close', () => {
+                // this means the underlying socket has been closed
+                log.debug('Client closed socket while streaming',
+                          { method: 'put', key, filePath,
+                            error: 'socket closed' });
```

Comment:
`error` should be an error object with message attribute to not mess up indexation. I suggest either wrapping it in an error object or calling the attribute name `errorMessage`

Reply:
Is it fine if I remove the error: field ? because there is no error in fact.